src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    private int aside;
+    private int bside;
+
+    public Rectangle(int aside, int bside, String color) {
+        this.aside = aside;
+        this.bside = bside;
+        setColor(color);
+        setFigureType("reactangle");
+    }
+
+    @Override
+    public void calculateArea() {
+        setArea(aside * bside);
+    }
+
+    @Override
+    public String toString() {
+        return super.toString()
+                + getFigureType()
+                + ", area: "
+                + getArea()
+                + " sq.units"
+                + ", a side: "
+                + aside
+                + " units, b side: "
+                + bside
+                + " units, color: "
+                + getColor();
+    }
```

Comment:
From the requirements:

```
To 'draw' means to print out all information about a figure using System.out.println() (you shouldn't override toString() method for this)
```

So, create an interface with one abstract method `draw()` and implement it in your `Figure`.